samples/bluetooth/tmap_peripheral/src/ccp_call_ctrl.c

```diff
@@ -139,6 +140,11 @@ int ccp_originate_call(void)
 	int err;
 	char uri[CONFIG_BT_TBS_MAX_URI_LENGTH];
 
+	if (strlen(remote_uri) == 0U) {
```

Comment:
Quite minor, but when you just want to know if the string is empty or not, `strlen()` is rather overkill since you end up iterating the entire string every time when you're actually only interested in the first byte. `if (str[0] == '\0')` is not the prettiest thing either, so it could be nice to have a helper for it.